app/views/overview.html

```diff
@@ -42,18 +42,18 @@
                    ng-if="service.metadata.labels.app || routesByService[service.metadata.name].length || childServicesByParent[service.metadata.name].length">
                 <overview-service-group></overview-service-group>
               </div>
-
-              <div row wrap>
+              
+              <div row wrap class="overview-standalone-service">
                 <div
                     ng-repeat="service in topLevelServices"
                     ng-if="!service.metadata.labels.app && !routesByService[service.metadata.name].length && !childServicesByParent[service.metadata.name].length"
                     class="standalone-service">
                   <overview-service-group></overview-service-group>
                 </div>
               </div>
-
+            
               <!-- Unserviced DCs, RCs, and pods -->
-              <div row wrap ng-if="(monopodsByService[''] | hashSize) > 0 || (deploymentConfigsByService[''] | hashSize) > 0 || (deploymentsByService[''] | hashSize) > 0">
+              <div row wrap ng-if="(monopodsByService[''] | hashSize) > 0 || (deploymentConfigsByService[''] | hashSize) > 0 || (deploymentsByService[''] | hashSize) > 0" class="overview-monopods">
```

Comment:
"monopod" is not really the right name since it can be other things, too. `class="unserviced-row"` maybe.